pkg/tools/log/logruslogger/formatter.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021 Doc.ai and/or its affiliates.
+// Copyright (c) 2024 Doc.ai and/or its affiliates.
```

Comment:
It's better to use a year range (2021-2024) for these cases.
Please check other modified files

Reply:
I have made updates to the headers.